doc/sphinx/proof-engine/ssreflect-proof-language.rst

```diff
@@ -4008,8 +4106,21 @@ definition.
       rewrite /=.
       unlock lid.
 
-.. tacn:: unlock {? @occ_switch } @ident
-   :name: unlock
+.. tacn:: unlock {* {? %{ @clear_switch %} } @term } {? @ssrclauses }
+   :name: unlock (ssreflect)
+
+   .. insertprodn ssrclauses lcpattern
+
+   .. prodn::
+      ssrclauses ::= in @ssrclausehyps {? %|- } {? * }
+      | in {| * | * %|- | %|- * }
+      ssrclausehyps ::= @gen_item {? {? , } @ssrclausehyps }
+      gen_item ::= @ssrclear
+      | {? @ } @ident
+      | ( @ident {? := @lcpattern } )
+      | (@ @ident := @lcpattern )
+      ssrclear ::= %{ {+ @ident } %}
+      lcpattern ::= {? Qed } @term
```

Comment:
Looking around I have the impression that `lconstr` was converted to `one_term` which is great. IMO `lcpattern` should become `one_cpattern`.

Reply:
`lconstr` in the mlg is `term` in the doc and `constr` in the mlg is `one_term`.  `cpattern` uses constr, so `one_pattern` would be consistent.  But not sure what `lcpattern` could become--there's already a `pattern` nonterminal, so we can't use that.  `ssr_pattern`?

Reply:
ok